zerver/webhooks/facebook/doc.md

```diff
@@ -0,0 +1,82 @@
+{!create-stream.md!}
+
+<!--
+{!create-bot-construct-url.md!}
+-->
+
+Next, on your {{ settings_html|safe }},
+[create a bot](/help/add-a-bot-or-integration) for
+{{ integration_display_name }}. Make sure that you select
+**Incoming webhook** as the **Bot type**:
+
+![](/static/images/help/bot_types.png)
+
+The API key for an incoming webhook bot cannot be used to read messages out
+of Zulip. Thus, using an incoming webhook bot lowers the security risk of
+exposing the bot's API key to a third-party service.
+
+Construct the URL for the {{ integration_display_name }}
+bot using the bot's API key and the desired stream name:
+
+`{{ api_url }}{{ integration_url }}?api_key=abcdefgh&stream={{ recommended_stream_name }}&token=sampletoken`
+
+Modify the parameters of the URL above, where `api_key` is the API key
+of your Zulip bot, and `stream` is the stream name you want the
+notifications sent to, and `token` will be used for server verification later,
+please decide your favorite character string.
+
+{!append-stream-name.md!}
+
+### Configuring the webhook
+
+Go and Sign In to the following URL: <https://developers.facebook.com/apps/>
+
+Next, click on **+ Add a New App** button.
+
+![](/static/images/integrations/facebook/001.png)
+
+Then, fill that form to create a new Facebook app.
+
+In `Display Name` column, enter the name as you can distinguishable your app.  
+And `Contact Email` column, enter the name as you can receive e-mail from Facebook.
+
+![](/static/images/integrations/facebook/002.png)
+
+Next, put your mouse cursor over **Webhooks** item, and click on **Set up** button.
+
+![](/static/images/integrations/facebook/003.png)
+
+Now, you can see the page like the image.  
+To choose a category for webhook, click on the drop down list.
+
+![](/static/images/integrations/facebook/004.png)
+
+This guide explains the procedure of subscribe to "feed" in the "user" category.
```

Comment:
Change this to `This guide explains how to subscribe to a "feed" in the **User** category.`